policybot/pkg/server/server.go

```diff
@@ -150,11 +151,16 @@ func serve(a *config.Args) error {
 		// stop the web server when we detect config changes, this causes a reload of everything
 		_ = srv.Shutdown(context.Background())
 	})
-
 	if err != nil {
 		return fmt.Errorf("unable to create config monitor: %v", err)
 	}
 
+	// TODO: do we want to have this bot for all repos or just istio/istio initially?
```

Comment:
We could extend the config format to make it possible to control which plugin is on for each repo.